debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/gccheck/CheckEngine.java

```diff
@@ -1463,3 +1463,4 @@ public boolean needVerifyOwnableSynchronizerConsistency()
 		return _needVerifyOwnableSynchronizerConsistency;
 	}
 }
+
```

Comment:
After removing the blank line, this file would be unchanged. Please remove it from the PR.